src/frontend/apps/standalone_site/src/features/Authentication/components/RenaterAuthenticator.spec.tsx

```diff
@@ -50,15 +52,59 @@ describe('<RenaterAuthenticator />', () => {
 
     userEvent.click(screen.getByRole('button', { name: /Open Drop/i }));
 
-    expect(
-      await screen.findByRole('option', { name: /Fake IdP 1/i }),
-    ).toBeInTheDocument();
-    expect(
-      screen.getByRole('option', { name: /Fake IdP 2/i }),
-    ).toBeInTheDocument();
-    expect(
-      screen.getByRole('option', { name: /Local accepting IdP/i }),
-    ).toBeInTheDocument();
+    const firstIdp = await screen.findByRole('option', { name: /Fake IdP 1/i });
+    const secondIdp = await screen.findByRole('option', {
```

Comment:
The first one need to be async but after you can durectly use `screen.getByRole`, they are in the DOM